packages/react-components/react-shared-contexts/src/AnnounceContext/AnnounceContext.ts

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+
+export type AnnounceOptions = {
+  alert?: boolean;
+  priority?: number;
+  batchId?: string;
+};
+
+/**
+ * @internal
+ */
+export type AnnounceContextValue = (message: string, options: AnnounceOptions) => void;
+
+/**
+ * @internal
+ */
+export const AnnounceContext = React.createContext<AnnounceContextValue | undefined>(undefined);
+
+/**
+ * @internal
+ */
+export const AnnounceProvider = AnnounceContext.Provider;
+
+export function useAnnounce(): AnnounceContextValue {
```

Comment:
haha oh no this means we can't call the hook to create the function `useAnnounce` 🙈. Not that this should change, it's following the naming schema of all the other contexts, but 😭, naming things i shard